bin/comment_with_dependency_diff

```diff
@@ -103,17 +106,34 @@ if [[ $sha != "$DEPENDENCY_TREE_CHECKSUM" ]]; then
 fi
 chmod +x dependency-tree-diff.jar
 
+JAR_DIR="$PLUGIN_ROOT/jar"
```

Comment:
Semantically speaking I'm not convinced that saying that the `/jar` dir is under `$PLUGIN_ROOT` makes sense—actually I don't think that having an env var named `PLUGIN_ROOT` but pointing to the root of a repo that is not a plugin makes sense as a var name in the first place…

```suggestion
REPO_ROOT=$( dirname "$( dirname "${BASH_SOURCE[0]}" )" )
JAR_DIR="$REPO_ROOT/jar"
```
Or
```suggestion
REPO_ROOT=$( git rev-parse --show-top-level )
JAR_DIR="$REPO_ROOT/jar"
```

Reply:
> (...) having an env var named PLUGIN_ROOT but pointing to the root of a repo that is not a plugin (...)

I might need a little more clarification: isn't `a8c-ci-toolkit-buildkite-plugin` a plugin?

Reply:
> I might need a little more clarification: isn't `a8c-ci-toolkit-buildkite-plugin` a plugin?

🤦  this is what I get from being tired and thinking I was reviewing a different repo…

Never mind, ignore my previous feedback then

Reply:
> I applied the suggestion anyway, as it doesn't require a diff in `environment`, which is neat! [4391e91](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/122/commits/4391e91c7648bb784089056eeff32c5590e96c52)

Yeah actually my suggestion was probably wrong, because once the Buildkite plugin will be installed by Buildkite in the agent there's a risk that `git rev-parse --show-code top-level` will resolve in the context of the app repo being built instead of the repo when the Buildkite plugin will be checked out … so there's a risk that my fix you applied will actually break I think.

Sorry for the wrong interpretation on my part; now that I realize that this code indeed lives in a Buildkite plugin, I think you should revert my suggestion and go back to how you implemented it initially with `PLUGIN_ROOT`, my bad …

Reply:
Gotcha, thank you for the help here 🙌 ! I double-checked if all works fine 
<img width="984" alt="image" src="https://github.com/user-attachments/assets/fa1bb42f-d37c-49e9-ac98-f3d5115809c0"> [673c7ab](https://github.com/Automattic/pocket-casts-android/pull/2857/commits/673c7abf58ba814211e3897c558ee902b01ae231)

Merging